Swiftfin/Views/SettingsView/CustomizeViewsSettings/Components/Sections/CustomizeHomeSection.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import SwiftUI
+
+extension CustomizeViewsSettings {
+    struct CustomizeHomeSection: View {
+
+        @Default(.Customization.Home.showRecentlyAdded)
+        private var showRecentlyAdded
+        @Default(.Customization.Home.maxNextUp)
+        private var maxNextUp
+        @Default(.Customization.Home.enableRewatching)
+        private var enableRewatching
+
+        var body: some View {
+            Section {
+                Toggle(L10n.showRecentlyAdded, isOn: $showRecentlyAdded)
+                Toggle(L10n.nextUpRewatch, isOn: $enableRewatching)
+                BasicStepper(
```

Comment:
I'm fine with this for now, but we will have to investigate another solution so that people aren't tapping/holding possibly hundreds of times. I would probably want to mirror the wheeled time interval picker, but there isn't one for months/weeks/days. We will probably have to take a look at my project below to make that for iOS but we'll have to think more for tvOS.

- https://github.com/LePips/MultiComponentPicker

That would change this implementation but we'll deal with that later.